packages/jest-cli/src/jest.js

```diff
@@ -111,6 +111,7 @@ const buildTestPathPatternInfo = argv => {
       input: argv._.join(' '),
       testPathPattern: argv._.join('|'),
       shouldTreatInputAsPattern: false,
+      findRelatedTests: argv.findRelatedTests,
```

Comment:
can you alphabetize this object? We try to keep things organized by alphabetizing things :)